cmd/minishift/cmd/delete.go

```diff
@@ -49,8 +51,12 @@ func runDelete(cmd *cobra.Command, args []string) {
 
 	util.ExitIfUndefined(api, constants.MachineName)
 
-	// Unregistration, do not allow to be skipped
-	unregisterHost(api, false)
+	if !forceMachineDeletion {
+		hasConfirmed := pkgUtil.AskForConfirmation("You are deleting the Minishift instance.")
```

Comment:
'the Minishift instance`... which one? the? or at least `this` or something else to indicate the active one?

Reply:
:+1:  for mentioning the current profile name instance here. It would be really helpful.

Reply:
I believe this was agreed... perhaps forgotten. Will add this next time to the first message of the PR when discussed on IRC in todo format

Reply:
Agree. Will fix it.